### PR TITLE
[MLIR][Lite] Update emit_quant_adaptor in PostQuantizePass

### PR DESCRIPTION
**Pull request by Vooblin**
Make separate quant-adaptors for inputs and outputs of model in PostQuantizePass.

### CODE REVIEW

File: tensorflow/compiler/mlir/lite/transforms/post_quantize.cc [hunk 1]
```diff
@@ -97,6 +102,28 @@ void RemoveQuantizationAdaptorOps(FuncOp func) {
     bb.eraseArgument(0);
   }
 
+  int num_return_operands = terminator->getNumOperands();
```
Comment (liufengdb) [inline tensorflow/compiler/mlir/lite/transforms/post_quantize.cc]: You can get the mlir::FunctionType and then query the results directly to get the result types.

File: tensorflow/compiler/mlir/lite/transforms/post_quantize.cc [hunk 2]
```diff
@@ -97,6 +102,28 @@ void RemoveQuantizationAdaptorOps(FuncOp func) {
...
+  auto new_func_type = builder.getFunctionType(input_types, output_types);
+  func.setType(new_func_type);
+}
+
+void RemoveOutputQuantizationAdaptorOps(FuncOp func) {
+  mlir::OpBuilder builder(func.getBody());
+  auto& bb = func.getBlocks().front();
+  auto* terminator = bb.getTerminator();
+
+  int num_args = bb.getNumArguments();
```
Comment (liufengdb) [inline tensorflow/compiler/mlir/lite/transforms/post_quantize.cc]: same here to get all the input types.

**Review by liufengdb — COMMENTED** [review]
Do you mind adding a FileCheck test to post_quantize.mlir?

**Comment by gbaned** [comment]
@Vooblin Can you please address reviewer comments and keep us posted? Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**